### PR TITLE
EVERREST-81 Correctly provide instances for test

### PR DESCRIPTION
**Pull request by skabashnyuk**

### CODE REVIEW

File: everrest-assured/src/test/java/org/everrest/assured/RemoteServiceDescriptorTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+
+package org.everrest.assured;
+
+
```
Comment (garagatyi) [inline everrest-assured/src/test/java/org/everrest/assured/RemoteServiceDescriptorTest.java]: remove extra lines

File: everrest-assured/src/test/java/org/everrest/assured/RemoteServiceDescriptorTest.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.UriInfo;
+
+import static com.jayway.restassured.RestAssured.when;
+import static org.testng.Assert.assertEquals;
+
+@Listeners(value = {EverrestJetty.class, MockitoTestNGListener.class})
+public class RemoteServiceDescriptorTest {
```
Comment (garagatyi) [inline everrest-assured/src/test/java/org/everrest/assured/RemoteServiceDescriptorTest.java]: Can you change name of test to something related to its purpose

**Comment by garagatyi** [comment]
PLease check year in licenses

**Comment by garagatyi** [comment]
everything else OK

**Comment by skabashnyuk** [comment]
PLease check year in licenses.
Will do that in separate PR. Its not related to this change

**Pull request merged**